src/app/shared/model/story.model.ts

```diff
@@ -160,10 +203,18 @@ export class StoryModel extends BaseModel implements HasUpload {
     this.updatedAt = new Date(this.doc['updatedAt']);
     this.publishedAt = this.doc['publishedAt'] ? new Date(this.doc['publishedAt']) : null;
     this.releasedAt = this.doc['releasedAt'] ? new Date(this.doc['releasedAt']) : null;
+
+    // count bytes of the html description
+    this.descriptionBytes = this.doc['descriptionMd'] || '';
```

Comment:
Do we need this assignment if we're assigning the following if/else?

Reply:
Whoops!  Don't know why i left that in there.